src/js/rx/components/SortMenu.jsx

```diff
@@ -1,16 +1,13 @@
 import PropTypes from 'prop-types';
-/* Component that lets a veteran sort by prescription name,
- * facility, or last submit date.
-*/
-
 import React from 'react';
 import _ from 'lodash';
 import classNames from 'classnames';
 
 const isDateAttribute = (option) => {
   return _.includes([
-    'refillSubmitDate',
+    'lastFillDate',
     'lastSubmitDate',
+    'refillSubmitDate',
     'refillDate'
```

Comment:
Two are actually used (`lastFillDate`, `lastSubmitDate`), and the other two just in case the raw attributes of the prescriptions objects are exposed.

`lastFillDate` and `lastSubmitDate` represent the formatted attributes that are displayed to the user. They're named in accord with the field names we also display to the user.